subprojects/ide-native/src/test/groovy/org/gradle/ide/cdt/model/CprojectSettingsSpec.groovy

```diff
@@ -1,58 +0,0 @@
-/*
- * Copyright 2011 the original author or authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.gradle.ide.cdt.model
-
-import org.gradle.api.Project
-import org.gradle.util.TestUtil
-import spock.lang.Ignore
-import spock.lang.Specification
-
-// very loose test, but I'm not expecting it to stay around
-@Ignore
```

Comment:
It has been ignored since 2014.  If somebody wants to test in this direction I think he should start again.